aries-site/src/examples/foundation/content-container-sizes/RadiusSizes.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { Box, FormField, Select } from 'grommet';
+import { useDesignTokens } from '../../../components/content/designTokenUtils';
+
+export const RadiusSizes = () => {
+  const [radius, setRadius] = React.useState('medium');
+  const { data } = useDesignTokens('semantic.radius');
```

Comment:
should we worry if Data will ever be undefined?

Reply:
done